src/JasperBus/JasperBusRegistry.cs

```diff
@@ -42,6 +44,16 @@ public ChannelExpression ListenForMessagesFrom(Uri uri)
             return new ChannelExpression(_feature.Channels, node);
         }
 
+        public void UseInMemoryTransport()
```

Comment:
Let's just have these enabled by default. It won't hurt anything *this* time if there are no channels for a transport.